### ISSUE TITLE
Set a default working directory initially for the user 

### ISSUE THREAD

**Issue by ltfschoen** [lukehaas/runjs#36]
I downloaded the .dmg file installed RunJS Version 1.2.0 (1.2.0) on macOS Mojave 10.14.0.

I tried importing the BigNumber.js library, so added this code:
```
const BigNumber = require('bignumber.js');
```

Then went to Action > Install NPM Modules, and entered 'bignumber.js' as a search criteria and added it.

Then I added the following:
```
const num = BigNumber(1);
console.log(num.div(2).toString());
```

But on the right it displayed:
```
TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received type undefined
```

I didn't realise what I had to do initially. Eventually I figured out that all I had to do was go to menu "Action > Set Working Directory", and chose a **random** working directory on my computer (i.e. ~/Documents), and then it displayed the output `0.5`.

Wouldn't it be better to onboard new users by automatically detecting their operating system and just choosing a typical default working directory already configured for them behind the scenes.

**Comment by lukehaas** [comment]
Installing node modules is not dependent on setting a working directory.
Not sure what happened here.
I'll investigate and see if I can recreate the issue.

**Comment by lukehaas** [comment]
This issue is now resolved in version 1.2.2
